openbsd-compat/bsd-misc.c

```diff
@@ -387,14 +387,6 @@ llabs(long long j)
 }
 #endif
 
-#ifndef HAVE_BZERO
-void
-bzero(void *b, size_t n)
-{
-	(void)memset(b, 0, n);
-}
-#endif
-
```

Comment:
bzero is still used in some places (eg the compat library, netcat in the tests) so this will break on platforms that don't have bzero.  It's also unrelated to the purported purpose of this commit so doesn't belong in this commit (in addition to being pointless churn).